src/components/ResponsiveSpacer.js

```diff
@@ -2,10 +2,17 @@ import React from 'react';
 import { createUseStyles } from 'react-jss';
 import classnames from 'classnames';
 
-const useStyles = createUseStyles(() => ({
+const useStyles = createUseStyles((theme) => ({
   root: {
-    paddingTop: 60,
-    paddingBottom: 60,
+    paddingTop: 24,
```

Comment:
This is unrelated but I noticed the vertical padding was a bit excessive on mobiles.